<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PulseWin</class>
 <widget class="QWidget" name="PulseWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PulseWin</string>
  </property>
  <widget class="QPushButton" name="pbPageFirst">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Page First</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPageGoto">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPageLast">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Page Last</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbScrollUp">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>50</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Scroll Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbScrollDown">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Scroll Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbScrollStop">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbFromFile">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>From File</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbFromBuffer">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>From Buffer</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAlg">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Alg</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
